src/main/java/forestry/apiculture/blocks/BlockBeeHives.java

```diff
@@ -197,5 +199,30 @@ public EnumBlockRenderType getRenderType(IBlockState state) {
 	public String getNameFromMeta(int meta) {
 		return HiveType.VALUES[meta].getName();
 	}
-
+	
+	@Override
+	public int getFlammability(IBlockAccess world, BlockPos pos, EnumFacing face)
+	{
+		if(!BiomeHelper.isBiomeHellish(world.getBiome(pos)))
+		{
+			return 5;
+		}
+		return 0;
+	}
+	
+	@Override
+	public boolean isFlammable(IBlockAccess world, BlockPos pos, EnumFacing face)
+	{
+		return !BiomeHelper.isBiomeHellish(world.getBiome(pos));
+	}
+	
+	@Override
+	public int getFireSpreadSpeed(IBlockAccess world, BlockPos pos, EnumFacing face)
+	{
+		if(!BiomeHelper.isBiomeHellish(world.getBiome(pos)))
```

Comment:
You don't have to test if the hive is in the nether, because the nether hive is from binnie and not from forestry.

Reply:
ah, I didn't realise binnie had their own hive material and block